kombu/connection.py

```diff
@@ -173,7 +173,7 @@ def __init__(self, hostname='localhost', userid=None,
                 # e.g. sqla+mysql://root:masterkey@localhost/
                 params['transport'], params['hostname'] = \
                     hostname.split('+', 1)
-                transport = self.uri_prefix = params['transport']
+                transport = self.uri_prefix = params['transport']  # unused
```

Comment:
I assume that `transport` is the variable not used elsewhere after this statement.

Reply:
@georgepsarakis If I revert this change, and add a `TODO` (or `FIXME`) note with my guess about indentation level of line 187, are you willing to merge the two other changes?

Reply:
For your previous comment, I do not think that this is the case, because this function is regularly used to parse URLs, so it would need a failing test to prove that something is wrong.
Perhaps change the line to `self.uri_prefix = params['transport']` ?

Reply:
I updated that.